### DISCUSSION TITLE
Any maintenance or updating plan for this tool in the future?

### DISCUSSION THREAD

**Discussion by Fan-HuaXD** [revapi/revapi#294] [Q&A] [answered]
We find your tool has great flexibility and powerful capability, so we want to use it as one of components of our own project. Meanwhile, we wonder:
1. if this tool will be maintained for a long-term?
2. if there is some updating plan in the future?

Thank you for your reply in advance!

**Comment by metlos** [comment, marked as answer]
Hi, as you probably noticed, there's not been any activity on this project for quite some time. I'm the sole maintainer of the project and don't have currently any time to develop it further. 

If you intend to provide fixes or new features to it, feel free to open pull requests and I will do my best to review them.
